### PR TITLE
Tiles!

### PR DESCRIPTION
**Pull request by hilburn**
The Guis still need a bit of work - ie speed indicators and progress bars, but I've gotta head off and thought you might want to have a play.

Current Issues:
No Orange side/top/bottom textures
The stamper active texture gets rejected for not being an animation
No progress bar/speed indicators in guis
Text label is clipping in Extruder

### CODE REVIEW

File: src/main/java/com/drullkus/thermalsmeltery/ThermalSmeltery.java [hunk 1]
```diff
@@ -22,6 +22,9 @@
 @Mod(modid = LibMisc.MOD_ID, name = LibMisc.MOD_NAME, dependencies = LibMisc.DEPENDENCIES)
 public class ThermalSmeltery
 {
+    @Mod.Instance(value = LibMisc.MOD_ID)
+    public static ThermalSmeltery instance = new ThermalSmeltery();
```
Comment (matthewprenger) [inline src/main/java/com/drullkus/thermalsmeltery/ThermalSmeltery.java:26]: No, should be:

``` java
@Mod.Instance(LibMisc.MOD_ID)
public static ThermalSmeltery instance;
```

FML will instantiate the field for us.

Comment (hilburn) [inline src/main/java/com/drullkus/thermalsmeltery/ThermalSmeltery.java:26]: It doesn't make a difference, FML loads it if it hasn't already been loaded - I personally prefer calling it myself

Comment (matthewprenger) [inline src/main/java/com/drullkus/thermalsmeltery/ThermalSmeltery.java:26]: I double checked how FML works, and it actually forcefully sets the instance field no matter if it is instantiated already or not. So while your method _does_ work, you are creating an instance of the Mod class that is never used.

**Comment by alexbegt** [comment]
I should be allowed to look at code before they are merged.

**Pull request merged**